docs/build.md

```diff
@@ -0,0 +1,66 @@
+# Electron Build Instructions
+
+## Common
+
+```bash
+git clone git@github.com:ubergrape/grape-electron.git
+cd grape-electron
+npm install
+npm run release
+```
+The result would be a installable (`dmg` and `app` or `msi` and `exe`) file in `./releases/` with version prefix that match version from `./app/package.json`
```

Comment:
I didn't fix all texts, just changed what needed, most of them a re old and with errors.